app/helpers/locale_helper.rb

```diff
@@ -40,6 +40,10 @@ def t_day(day)
     I18n.t("date.other.#{day}", default: '')
   end
 
+  def t_month(month)
```

Comment:
maybe we could raise and capture an error here if month number is out of range e.g. `fail unless month.between?(1,12)`?

Reply:
(or not an integer)

Reply:
It should handle an out-of-bounds index by returning a nil, and not sure that we've generally logged that level of data error..?